README.md

```diff
@@ -84,16 +84,20 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     League of Women Coders (formerly Ladies Who Code, est. June 2011) is a meetup that focuses on coding, hacking, and learning collaboratively. The group meets up at least once a month in New York City and Boston. The meetups focus on talks, technical questions and ideation, and occasionally feature female speakers.
 
+  - [RodAda Hacker](http://rodadahacker.org/)  - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
+
+    The RodAda Hacker is a Brazilian network that provides collaborative workshops of programming specially designed for girls and women. It is a space for who wants to dream, build awesome projects and rethink the technology. The name of the project is a tribute to Ada Lovelace.
```

Comment:
* 'It is a space for who wants to dream, build awesome ' can be 'It is a space for those who want to dream, build awesome ...'

* 'The name of the project is a tribute to Ada Lovelace.' this does not add any special information, and it quite obvious from the name (and every third item in the last is named after her)

Reply:
I'll change it. =]